docs/keycloak/manual-client-secret-setup-pl.md

```
@@ -0,0 +1,92 @@
+> 📘 Ten poradnik jest również dostępny w [🇬🇧 wersji angielskiej](manual-client-secret-setup.md)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)